include/grpc/impl/codegen/grpc_types.h

```diff
@@ -726,6 +726,10 @@ typedef struct grpc_experimental_completion_queue_functor {
       operation succeeded (non-zero) or failed (zero) */
   void (*functor_run)(struct grpc_experimental_completion_queue_functor*, int);
 
+  /** The inlineable member specifies whether this functor can be run inline.
+      This should only be used for trivial internally-defined functors. */
+  int inlineable;
```

Comment:
non-blocking comment - Do we foresee this field being used by users, i.e., will we need to provide a knob to users to set this?

Reply:
Addressed on reviewable